lib/memory_profiler/helpers.rb

```diff
@@ -1,10 +1,14 @@
+require 'digest'
+
 module MemoryProfiler
   class Helpers
+    STRING_MAX = 199 # 200 chars
```

Comment:
Might see what the effects of switching this to 200, and then changing the `[]` invocation in `string_summary` to not use a range, but the `[index], [length]` arguments.  Probably is a bit clearer, but that said, some of this is a relic from when I was testing this with a "cache file" based approach.